src/ledger/signer.ts

```diff
@@ -0,0 +1,130 @@
+import { BigNumber } from "bignumber.js";
```

Comment:
This should be at the `src` level, not in `ledger`

Reply:
Do you think it should go in the `wallet` repository?

Reply:
That seems reasonable as well.